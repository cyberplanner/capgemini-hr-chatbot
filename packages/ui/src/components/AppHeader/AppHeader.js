import React, { Component } from "react";
import styled from "styled-components";

import octo from "./Octopus-128.png";

const Header = styled.header`
  background-color: #455963;
  height: 60px;
  color: #ebebec;
  border-bottom: 2px solid rgba(100, 100, 100, 0.5);
  text-align: center;
  display: flex;
  align-items: center;
  padding: 0em 2em;
  & > img {
    height: 100%;
    width: auto;
  }
`;

class AppHeader extends Component {
  render() {
    return (
      <Header>
        <img src={octo} />
        <h3>Ubiqutious Octo Pancake</h3>
      </Header>
    );
  }
}

export default AppHeader;
