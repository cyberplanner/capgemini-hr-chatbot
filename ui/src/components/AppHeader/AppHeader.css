.App-logo {
    animation: App-logo-spin infinite 7s linear;
    height: 60px;
    margin: 0 30px;
}

.App-header {
    background-color: #455963;
    height: 60px;
    color: #EBEBEC;
    border-bottom: 2px solid rgba(100,100,100,0.5);
    text-align: center;
    display: flex;
    align-items: center;
}

@keyframes App-logo-spin {
    from {
        transform: rotateY(0deg);
    }
    to {
        transform: rotateY(360deg);
    }
}
