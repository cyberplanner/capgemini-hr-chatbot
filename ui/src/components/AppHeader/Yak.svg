<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="512px" height="512px" viewBox="0 0 512 512" enable-background="new 0 0 512 512" xml:space="preserve">
<pattern  width="14.036" height="29.766" patternUnits="userSpaceOnUse" id="Unnamed_Pattern" viewBox="0 -29.92 14.036 29.766" overflow="visible">
	<g>
		<polygon fill="none" points="0,-0.153 14.036,-0.153 14.036,-29.92 0,-29.92 		"/>
		<path fill="#010101" d="M14.036-29.88c-4.31-0.317-8.84,1.248-12.248,4.656c-0.026,0.027-0.053,0.055-0.081,0.081
			c1.965,2.513,2.999,5.636,2.919,8.983c-0.094,3.973-1.73,7.775-4.607,10.702C0.012-5.45,0.006-5.446,0-5.44
			c0.368,0.473,0.758,0.93,1.191,1.355C3.937-1.394,7.639,0,11.616-0.167c0.818-0.033,1.625-0.146,2.42-0.307V-29.88z"/>
	</g>
</pattern>
<g>
	<path d="M327.355,222.618c-3.533,0.033-6.844,1.439-9.32,3.961c-2.477,2.522-3.822,5.856-3.789,9.391
		c0.065,7.255,5.989,13.112,13.23,13.112c0.033,0,0.066,0,0.1,0c0.334-2.91,1.605-5.612,3.69-7.736
		c2.477-2.522,5.787-3.928,9.32-3.961c0.008,0,0.015,0.001,0.021,0.001c0.063-0.546,0.104-1.097,0.1-1.656
		c-0.032-3.534-1.438-6.845-3.96-9.321C334.228,223.932,330.913,222.566,327.355,222.618z"/>
	<path d="M182.177,222.618c-3.533,0.033-6.844,1.439-9.32,3.961c-2.477,2.522-3.822,5.856-3.789,9.391
		c0.065,7.255,5.989,13.112,13.23,13.112c0.033,0,0.066,0,0.1,0c0.334-2.91,1.605-5.612,3.69-7.736
		c2.477-2.522,5.787-3.928,9.32-3.961c0.008,0,0.015,0.001,0.021,0.001c0.063-0.546,0.104-1.097,0.1-1.656
		c-0.032-3.534-1.438-6.845-3.96-9.321C189.049,223.932,185.734,222.566,182.177,222.618z"/>
	<path d="M381.609,222.6l2.979-27.329l0,0c0.237-2.405,0.456-4.707,0.656-6.885c0.059-0.634,0.115-1.257,0.17-1.871
		c0.656-7.301,1.094-13.156,1.315-17.182c18.962-4.959,76.773-21.201,93.342-38.902c9.42-10.064,11.229-26.788,5.382-49.706
		c-4.098-16.061-10.504-29.371-10.774-29.93c-1.308-2.7-4.42-4.286-7.624-3.893c-3.209,0.394-5.746,2.673-6.225,5.592
		c-1.694,10.329-7.545,39.527-14.14,50.395c-0.175,0.231-4.709,5.718-42.077,11.285c-19.951,2.972-39.036,4.426-39.229,4.441
		c-0.679,0.051-1.324,0.191-1.932,0.397c-5.479-4.371-11.726-8.075-18.555-10.828c-23.025-9.28-61.037-12.586-88.935-12.586
		c-27.897,0-65.909,3.307-88.935,12.586c-6.822,2.75-13.061,6.449-18.535,10.812c-0.594-0.197-1.222-0.332-1.882-0.381
		c-0.189-0.015-19.276-1.469-39.228-4.44c-37.365-5.567-41.901-11.054-42.045-11.238c-6.625-10.915-12.475-40.112-14.17-50.442
		c-0.479-2.919-3.015-5.199-6.225-5.593c-3.208-0.395-6.314,1.193-7.623,3.894c-0.271,0.559-6.677,13.87-10.774,29.93
		c-5.848,22.918-4.037,39.642,5.381,49.707c16.871,18.024,75.753,34.635,93.293,39.276c0.343,6.077,1.155,16.077,2.44,28.777
		c-0.022,0.009-0.044,0.015-0.066,0.024l0.028,0.258c0.688-0.05,1.379-0.187,2.081-0.448l1.245,13.339
		c-0.64,0.151-1.27,0.252-1.895,0.326l0.009,0.083c0.015-0.003,0.029-0.005,0.043-0.008c0.804,7.22,1.729,15.017,2.776,23.242
		c-0.037,0.043-0.071,0.09-0.107,0.134l0.313,2.402c0.404-0.582,0.786-1.167,1.141-1.751l1.605,17.189
		c-0.163,0.149-0.332,0.288-0.497,0.434c8.503,61.004,22.888,137.275,43.406,176.783c0.118,0.225,0.258,0.434,0.399,0.643
		c-0.824,8.384,1.87,16.715,7.732,23.433c6.993,8.015,17.422,12.525,28.599,12.466l85.063,0.125c0.003,0,0.006,0,0.012,0
		c0.021,0,0.042,0,0.065,0c11.232-0.094,21.593-4.81,28.423-12.939c6.142-7.31,8.605-16.41,6.993-25.376
		c19.058-37.932,32.74-107.478,41.251-165.909c0.075-0.514,0.149-1.026,0.224-1.538c0.184-1.269,0.362-2.529,0.541-3.787
		l4.561-34.98C381.607,222.619,381.608,222.609,381.609,222.6z M125.482,155.942c-24.427-6.699-70.493-21.306-82.488-34.123
		c-6.058-6.472-6.882-19.934-2.325-37.906c0.398-1.569,0.822-3.113,1.262-4.623c2.719,10.986,6.383,22.99,10.679,30.068
		c3.113,5.13,11.802,11.788,51.688,17.796c12.5,1.883,24.512,3.169,32.298,3.91C130.738,138.701,126.837,147.263,125.482,155.942z
		 M316.487,446.149c-4.08,4.856-10.265,7.677-16.969,7.742l-85.1-0.123c-6.715,0.043-12.976-2.662-17.16-7.459
		c-4.185-4.797-5.683-10.965-4.112-16.925l5.652-21.445c0.969-3.674,8.668-20.915,27.664-20.589l0.015-0.008l0.592,0.001
		c0.003-0.004,0.013-0.014,0.017-0.018l56.217,0.32c1.386-0.202,2.802-0.327,4.257-0.327c9.976,0,18.54,4.966,22.341,12.073
		c1.921,2.525,3.409,5.352,4.283,8.42l6.081,21.349C321.952,435.093,320.577,441.286,316.487,446.149z M372.339,165.475
		c-0.257,12.628-4.883,61.33-13.204,113.575c-6.408,40.24-16.511,92.113-30.183,127.844l-0.688-2.416
		c-4.98-17.484-22.813-30.086-42.5-30.086c-0.104,0-0.214,0-0.321,0.001l-58.282-0.333c-0.145-0.006-0.292-0.006-0.439-0.003
		l-0.513-0.003c-0.038,0-0.074,0-0.111,0c-19.665,0.162-36.708,12.821-41.442,30.783l-0.997,3.784
		c-14.014-35.772-24.349-88.675-30.861-129.571c-8.458-53.109-13.099-102.557-13.215-114.173c-0.001-0.21-0.017-0.418-0.04-0.623
		c-0.928-17.046,13.311-35.854,33.372-43.94c17.263-6.958,49.863-11.454,83.052-11.454c33.19,0,65.789,4.497,83.054,11.454
		c19.451,7.841,33.429,25.762,33.414,42.383C372.192,163.585,372.151,164.527,372.339,165.475z M375.34,131.072
		c7.784-0.741,19.827-2.029,32.361-3.917c39.885-6.008,48.575-12.666,51.688-17.795c4.297-7.079,7.961-19.083,10.68-30.068
		c0.439,1.508,0.863,3.054,1.262,4.623c4.558,17.971,3.733,31.434-2.326,37.905c-11.853,12.665-57.635,27.021-82.61,33.781
		C384.993,147.042,381.118,138.607,375.34,131.072z"/>
	<path d="M226.962,402.174c-1.463-1.33-2.961-2.691-4.008-3.89h-0.001l-0.03-0.034c-0.02-0.022-0.038-0.045-0.057-0.068
		l-2.312-2.714c-5.064,3.865-10.315,7.776-9.764,8.654c1.869,2.981,4.608,5.469,7.257,7.874c1.9,1.725,5.437,4.937,5.6,6.178
		c-0.002,0.012-0.357,1.333-3.808,3.736c-3.006,2.096-3.745,6.229-1.651,9.234c1.289,1.851,3.351,2.841,5.448,2.841
		c1.309,0,2.631-0.385,3.786-1.191c6.041-4.209,9.136-8.705,9.46-13.744C237.36,411.613,231.596,406.379,226.962,402.174z"/>
	<path d="M294.33,395.468l-2.313,2.714c-0.018,0.023-0.036,0.046-0.056,0.068l-0.03,0.034h-0.001c-1.048,1.198-2.545,2.56-4.01,3.89
		c-4.633,4.205-10.397,9.439-9.919,16.876c0.325,5.039,3.419,9.535,9.46,13.744c1.155,0.807,2.477,1.191,3.786,1.191
		c2.097,0,4.158-0.99,5.448-2.841c2.094-3.005,1.355-7.139-1.652-9.234c-3.449-2.403-3.805-3.725-3.806-3.736
		c0.163-1.241,3.698-4.453,5.6-6.178c2.648-2.405,5.388-4.893,7.256-7.874C304.645,403.244,299.394,399.333,294.33,395.468z"/>
</g>
<g>
	<path d="M105.319,249.051c-7.434-1.851-15.658-6.404-22.562-12.493c-6.819-6.015-9.617-11.04-9.967-12.668
		c0.125-2.567,3.686-8.86,13.217-16.254c8.658-6.716,16.991-10.341,19.677-10.374c0.006,0,0.013,0,0.019,0
		c1.699,0,4.599,2.889,7.159,5.439c4.835,4.818,11.176,11.096,18.832,8.914l-1.029-13.339c-3.827,1.72-7.271-1.38-12.282-6.371
		c-3.932-3.918-8.019-7.95-12.785-7.908c-5.793,0.073-16.563,6.114-24.208,12.044c-12.076,9.366-18.168,18.828-18.108,28.123
		c0.084,12.972,17.725,29.214,33.505,35.679c5.609,2.3,10.954,3.43,15.94,3.429c8.65-0.002,16.204-3.427,22.178-10.039
		l-1.327-17.188C129.394,244.381,121.15,252.99,105.319,249.051z"/>
	<g>
		<pattern  id="SVGID_1_" xlink:href="#Unnamed_Pattern" patternTransform="matrix(-0.6533 0 0 1 -13106.7783 2.6387)">
		</pattern>
		<path fill="#010101" stroke="url(#SVGID_1_)" stroke-miterlimit="10" d="M69.569,209.613c2.815-0.316,5.775,1.248,8.001,4.656
			c0.017,0.027,0.035,0.055,0.053,0.082c-1.284,2.512-1.959,5.635-1.907,8.982c0.062,3.973,1.131,7.775,3.01,10.703
			c0.004,0.006,0.008,0.01,0.012,0.016c-0.24,0.473-0.495,0.93-0.778,1.355c-1.793,2.691-4.211,4.086-6.81,3.918
			c-0.535-0.033-1.062-0.146-1.581-0.307V209.613z"/>
	</g>
</g>
<g>
	<path d="M404.531,249.051c7.434-1.851,15.658-6.404,22.561-12.493c6.819-6.015,9.617-11.04,9.968-12.668
		c-0.124-2.567-3.687-8.86-13.218-16.254c-8.658-6.716-16.99-10.341-19.676-10.374c-0.007,0-0.014,0-0.02,0
		c-1.698,0-4.599,2.889-7.158,5.439c-4.836,4.818-11.176,11.096-18.832,8.914l1.028-13.339c3.827,1.72,7.271-1.38,12.282-6.371
		c3.932-3.918,8.018-7.95,12.785-7.908c5.793,0.073,16.562,6.114,24.207,12.044c12.076,9.366,18.168,18.828,18.107,28.123
		c-0.084,12.972-17.725,29.214-33.504,35.679c-5.609,2.3-10.955,3.43-15.94,3.429c-8.649-0.002-16.204-3.427-22.178-10.039
		l1.327-17.188C380.457,244.381,388.699,252.99,404.531,249.051z"/>
	<g>
		<pattern  id="SVGID_2_" xlink:href="#Unnamed_Pattern" patternTransform="matrix(0.6533 0 0 1 -2760.1255 2.6387)">
		</pattern>
		<path fill="#010101" stroke="url(#SVGID_2_)" stroke-miterlimit="10" d="M440.281,209.613c-2.815-0.316-5.775,1.248-8.001,4.656
			c-0.017,0.027-0.034,0.055-0.054,0.082c1.284,2.512,1.96,5.635,1.908,8.982c-0.062,3.973-1.131,7.775-3.011,10.703
			c-0.005,0.006-0.009,0.01-0.012,0.016c0.24,0.473,0.495,0.93,0.778,1.355c1.793,2.691,4.211,4.086,6.811,3.918
			c0.534-0.033,1.061-0.146,1.58-0.307V209.613z"/>
	</g>
</g>
</svg>
