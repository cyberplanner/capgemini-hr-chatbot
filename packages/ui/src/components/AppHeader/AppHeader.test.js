import React from "react";
import AppHeader from "./AppHeader";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const tree = renderer.create(<AppHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
