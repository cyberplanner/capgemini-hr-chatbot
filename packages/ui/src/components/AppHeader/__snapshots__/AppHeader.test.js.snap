// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<header
  className="sc-bdVaJa ihtZGL"
>
  <img
    src="Octopus-128.png"
  />
  <h3>
    Ubiqutious Octo Pancake
  </h3>
</header>
`;
